Admin/AbstractAdmin.php

```diff
@@ -1053,6 +1053,27 @@ public function isCurrentRoute($name, $adminCode = null)
     }
 
     /**
+     * Checks if a filter is set.
+     *
+     * @param string $name
+     *
+     * @return bool
+     */
+    public function isDefaultFilter($name)
+    {
+        $filter = $this->getFilterParameters();
+
+        if (!array_key_exists($name, $filter)) {
+            return false;
+        }
+
+        $default = array_key_exists($name, $this->datagridValues) ? $this->datagridValues[$name]['value'] : null;
+        $value = $filter[$name]['value'];
+
+        return $value != $default;
```

Comment:
I don't get it… this method will return false if `$default` and `$value` are the same. It's literally the opposite of what the name suggests. What am I missing?